cavern/VERSION

```diff
@@ -1,6 +1,6 @@
 ## deployable containers have a semantic and build tag
 # semantic version tag: major.minor
 # build version tag: timestamp
-VER=0.6.3
+VER=0.6.4
```

Comment:
currently published cavern image is 0.6.2

Reply:
One last time hopefully, updated.